app/views/ExposureHistory/Calendar.tsx

```diff
@@ -36,14 +40,15 @@ const Calendar = ({
     return (
       <View style={styles.calendarRow}>
         {week.map((datum: ExposureDatum) => {
+          const isSelected = datum.date === selectedDatum?.date;
+
           return (
             <TouchableOpacity
               key={`calendar-day-${datum.date}`}
-              onPress={() => onSelectDate(datum)}>
+              onPress={() => onSelectDate(datum)}
+              disabled={isAfterToday(datum.date)}>
```

Comment:
so this is a minor, extremely pedantic, point of view, but thought you might want to geek out on pedantic coding points of view like the good ol days.

Using the disabled prop here is mildly more imperative, rather than declarative, approach to this feature than putting the conditional in the `onSelectDate` function.  That is, the high level declaration of the behavior you want is 'when a user taps a date, if that date is not in the future, move the circle to that date. So by adding isAfterToday(datum.date) to the disabled prop you are indicating the `how` to do this rather than the `what` you want.

Another way to think about it is you have two places where the 'what do I do when the user taps a thing' logic happens rather than one.

Of course, this is an extremely silly of thing to split hairs on, and i wouldnt change anything in this commit.

Reply:
I love this stuff haha. Miss those days. I was actually wondering if this was the best place to house this logic. I guess it’s kinda pedantic in a way but I think it matters at scale. You do this 10 times and it adds up. I may take a look at this in the morning, maybe we could pair on it?

Reply:
totally! hit me up and we can take a look.